mysite/settings/base.py

```diff
@@ -1,23 +1,427 @@
-from __future__ import unicode_literals
-from os.path import join
+import os
+import sys
+from os.path import join, abspath, dirname
 
+# PATH vars
 
-from .conf import get_settings
+here = lambda *x: join(abspath(dirname(__file__)), *x)
+PROJECT_ROOT = here("..")
+root = lambda *x: abspath(join(abspath(PROJECT_ROOT), *x))
+REPO_ROOT = lambda *x: join(abspath(here("../..")), *x)
+ROOT = REPO_ROOT
+BASE_DIR = root('..')
 
-globals().update(get_settings('general.yml'))
 
-# I'm not a big fan of having a local_settings.py file outside version
-# control but some partners have asked for this, so this code tries to
-# load that file if it's present. (This use Matthew's suggestion for
-# allowing local settings overrides with both Python 2 and Python 3;
-# this uses exec rather than import so that the local settings can
-# modify existing values rather than just overwriting them.)
-LOCAL_SETTINGS = join(BASE_DIR, 'mysite', 'settings', 'local_settings.py')
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '1$s^ggnnc16*_9=a^5yv3jr4pw3a=f##c#fc!ewc&i5n^q88l%'
```

Comment:
You might be thinking about https://lgtm.com – I've set up YNR on that now, but it'll take some time to load, and I expect it will come up with…a lot of stuff as the code base is quite big.

I've fixed the `SECRET_KEY` one too.